cmd/config-migrate.go

```diff
@@ -2447,16 +2448,16 @@ func migrateConfigToMinioSys(objAPI ObjectLayer) (err error) {
 	defer objLock.Unlock()
 
 	// Verify if backend already has the file (after holding lock)
-	if err = checkConfig(context.Background(), objAPI, configFile); err != errConfigNotFound {
+	if err = checkConfig(context.Background(), objAPI, configFile); err != errFileNotFound {
 		return err
-	} // if errConfigNotFound proceed to migrate..
+	} // if errFileNotFound proceed to migrate..
 
 	var configFiles = []string{
 		getConfigFile(),
 		getConfigFile() + ".deprecated",
 		configFile,
 	}
-	var config = &serverConfig{}
+	var config = &serverConfigV27{}
```

Comment:
why V27 ? shouldn't it be the latest one ? (33)

Reply:
Because V27 is when we went to backend `.config.json` not V33